metsim/disaggregate.py

```diff
@@ -363,16 +363,20 @@ def shortwave(sw_rad: pd.Series, daylength: pd.Series, day_of_year: pd.Series,
     tinystep[inds] += cnst.HOURS_PER_DAY * tiny_step_per_hour
     inds = np.array(tinystep > (cnst.HOURS_PER_DAY * tiny_step_per_hour-1))
     tinystep[inds] -= (cnst.HOURS_PER_DAY * tiny_step_per_hour)
+    tinystep = np.array(tinystep).astype(np.int32)
```

Comment:
can you use `np.asarray()` instead here (and above). You can also specify the dtype there too (i.e. `np.asarray(tinystep, dtype=np.in32)`